packages/netlify-cms-core/src/reducers/mediaLibrary.js

```diff
@@ -193,4 +187,33 @@ const mediaLibrary = (state = Map(defaultState), action) => {
   }
 };
 
+export function selectMediaFiles(state) {
+  const { mediaLibrary, entryDraft } = state;
+  const workflowDraft = selectEditingWorkflowDraft(state);
+  const integration = selectIntegration(state, null, 'assetStore');
+
+  let files;
+  if (workflowDraft && !integration) {
```

Comment:
Select media files either from the draft or from the "global" media folder

Reply:
Nice.